src/utils/EN.js

```diff
@@ -111,12 +111,12 @@ exports.ORDINAL_WORDS_PATTERN = '(?:'
     + ')';
 
 var TIME_UNIT = 
-    '(' + exports.INTEGER_WORDS_PATTERN + '|[0-9]+|an?(?:\\s*few)?|half(?:\\s*an?)?)\\s*' +
+    '(' + exports.INTEGER_WORDS_PATTERN + '|[0-9]+|[0-9]+\.[0-9]+|an?(?:\\s*few)?|half(?:\\s*an?)?)\\s*' +
```

Comment:
This and the `parseFloat` were what was needed to support fractional time chunks